hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/compactions/Compactor.java

```diff
@@ -464,7 +464,6 @@ protected boolean performCompaction(FileDetails fd, InternalScanner scanner, Cel
             lastCleanCell = null;
             lastCleanCellSeqId = 0;
           }
-          writer.append(c);
```

Comment:
Is this change related?

Reply:
DualFIleWriter works with both append() and the new appendAll(). I added the appendAll() for the compaction performance reasons as appendAll() eliminates the row key comparison for the cells of the same row (except the first cell).

Reply:
Let's not mix different things up? I'm not sure if there are other side effects about this change...